examples/async/async.bal

```diff
@@ -40,28 +41,27 @@ public function main() {
     }
     io:println(f3.isDone());
 
-    // Asynchronously call the below functions and wait for either of them to finish
     future<int> f4 = start square(20);
     future<string> f5 = start greet("Bert");
-    // You can wait for either of the above asynchronous functions to finish
+    // You can wait for either of the asynchronous functions to finish.
+    // Here `f4` will finish before `f5` since a `runtime:sleep` is 
+    // added to the `greet()` function to delay its execution. The 
+    // asynchronous function that finishes first will be taken as the result. 
```

Comment:
Shall we modify this to something like "The value returned by the asynchronous function..."